packages/phone-number-privacy/signer/test/end-to-end/get-blinded-sig.test.ts

```diff
@@ -128,25 +119,14 @@ describe('Running against a deployed service', () => {
 
     it('Returns sig when querying succeeds with used request', async () => {
```

Comment:
Does this test assume that the test above runs first? It looks like the same test otherwise.

Reply:
Good call out. I've updated it to test for the new behavior